blockchain.go

```diff
@@ -490,8 +490,24 @@ func configureBootstrapProcedure(conf config, flowAccountKey flowgo.AccountPubli
 		)
 	}
 	if conf.TransactionFeesEnabled {
+		// This enables variable transaction fees AND execution effort metering
+		// as described in Variable Transaction Fees: Execution Effort FLIP: https://github.com/onflow/flow/pull/753)
+		// TODO: In the future this should be an injectable parameter. For now this is hard coded
+		// as this is the first iteration of variable execution fees.
 		options = append(options,
-			fvm.WithTransactionFee(fvm.DefaultTransactionFees),
+			fvm.WithTransactionFee(fvm.BootstrapProcedureFeeParameters{
+				SurgeFactor:         cadence.UFix64(100_000_000), // 1.0
+				InclusionEffortCost: cadence.UFix64(100),         // 1E-6
+				ExecutionEffortCost: cadence.UFix64(499_000_000), // 4.99
+			}),
+			fvm.WithExecutionEffortWeights(map[common.ComputationKind]uint64{
+				1001: 1569,
```

Comment:
how come these key values are not defined as constants in the FVM? wouldn't that way be clearer what the computation type is?